.gitignore

```diff
@@ -10,3 +10,6 @@ coverage
 deployment/*.retry
 
 /yarn-error.log
+
+test/image/build
```

Comment:
Could you use 

https://github.com/plotly/orca/tree/master/build

instead, like our other tests?

Reply:
Done in 697068a